rpcs3/Emu/Cell/Modules/cellRtc.cpp

```diff
@@ -842,40 +817,18 @@ CellRtcDateTime tick_to_date_time(u64 tick)
 
 u64 date_time_to_tick(CellRtcDateTime date_time)
 {
-	const auto get_days_in_year = [](u16 year, u16 months) -> u64
-	{
-		const bool leap = is_leap_year(year);
-		u64 days = 0;
-		for (u16 m = 0; m < months; m++)
-		{
-			days += DAYS_IN_MONTH[m + (leap * 12)];
-		}
-		return days;
-	};
+	const u32 days_in_previous_years =
+		date_time.year * 365
+		+ (date_time.year + 3) / 4
+		- (date_time.year + 99) / 100
+		+ (date_time.year + 399) / 400
+		- 366;
 
-	u64 days = 0;
+	ensure(date_time.month - 1u < 12u); // Not checked on LLE
```

Comment:
it is what it is then, don't assert it.

Reply:
I'd just min month to 12 and log a warning